### PR TITLE
feature/CU-vhcbwu-Write-Docs-Hotel-Booking

### PR DESCRIPTION
**Pull request by mcclymont-k**
**DESCRIPTION**
Docs for the hotel booking request and response.

**CHANGES**
1. Added hotel book request.
2. Added hotel book response.

**TESTING**
1. Run the docs and compare them with the models in the MCFS PR here: https://github.com/trip-ninja-inc/trip_ninja_api/pull/1230

### CODE REVIEW

**Comment by trip-ninja-inc** [comment]
Task linked: [CU-vhcbwu Write Docs Hotel Booking](https://app.clickup.com/t/vhcbwu)

File: tn_api_hotel.yml [hunk 1]
```diff
@@ -1080,3 +1121,130 @@ components:
...
+                enum: [F, M]
+              passenger_type:
+                description: Type of passenger.
+                type: string
+                example: "ADT"
+                enum: [ADT, STU, CHD, INF]
+              email:
+                description: Passenger's email address.
+                type: string
+                example: "andres.collar@tripninja.io"
```
Comment (andrescollart) [inline tn_api_hotel.yml]: hmm typo on my email!

File: tn_api_hotel.yml [hunk 2]
```diff
@@ -1080,3 +1121,130 @@ components:
                 description: The full text of the ruledescription
                 example: 'non-refundable rate cancellation permitted up to 1days before arrival'
 
+    HotelBookRequest:
+      required:
+        - super_trip_id
+        - passengers
+        - payment_info
```
Comment (andrescollart) [inline tn_api_hotel.yml:1128]: This isn't required is it? We can book one and have the payment be through 'cash' right? Although there's the guarantee bit sometimes - but not always.

**Review by andrescollart — CHANGES_REQUESTED** [review]
Verify the requirement for payment_info

File: tn_api_hotel.yml (hunk 1 above)
Comment (mcclymont-k) [reply tn_api_hotel.yml]: I copied it from somewhere else in the docs -- does it matter, just an example?

File: tn_api_hotel.yml (hunk 2 above)
Comment (mcclymont-k) [reply tn_api_hotel.yml:1128]: We talked about this

**Review by andrescollart — APPROVED** [review]

**Pull request merged**